packages/dialog/src/dialog-wrapper.ts

```diff
@@ -138,7 +138,11 @@ export class DialogWrapper extends LitElement {
             >
                 ${this.hero
                     ? html`
-                          <img src="${this.hero}" slot="hero" />
+                          <img
+                              src="${this.hero}"
+                              slot="hero"
+                              aria-hidden="true"
```

Comment:
This is cheating! Currently, Spectrum CSS uses a background image for this, which is very close to the same level of cheating, so I wasn't sure what to do about applying an `alt` tag here. Should we use `aria-hidden`? Should we reapply the `headline` text? Should we surface an additional attribute? Should we move to a background image? I'm not sure the right path for both functionality and accessibility here...

Reply:
As discussed offline, the second alternate that leverages the `hero-label` attribute makes the most sense to me. 

Both the `headline` and `hero-label` alternatives align with how we're handling the alt tag of shadowDOM images in other components, such as `Avatar`. I prefer the `hero-label` implementation, since the headline of the dialogWrapper doesn't always align with the contents of the hero image

Reply:
I've updated `sp-dialog-wrapper` to include `hero-label` attribute which applies to the `alt`  attribute to the `<img />` when available. When unavailable, the `aria-hidden="true"` practice is maintained.